packages/components/Tabs/src/Tabs.windows.tsx

```diff
@@ -0,0 +1,177 @@
+    });
+
+    const onKeyDown = (ev: any) => {
+      if (ev.nativeEvent.key === 'ArrowRight' || ev.nativeEvent.key === 'ArrowLeft') {
+        const length = state.info.enabledKeys.length;
+        const currTabItemIndex = state.info.enabledKeys.findIndex(x => x == state.context.selectedKey)
+        let newCurrTabItemIndex;
+        if (ev.nativeEvent.key === 'ArrowRight') {
+          if (isCircularNavigation || !(currTabItemIndex + 1 == length)) {
+            newCurrTabItemIndex = (currTabItemIndex + 1) % length;
+            state.context.selectedKey = state.info.enabledKeys[newCurrTabItemIndex];
+            data.onKeySelect(state.context.selectedKey);
+          }
+        }
+        else {
+          if (isCircularNavigation || !(currTabItemIndex == 0)) {
+            newCurrTabItemIndex = (currTabItemIndex - 1 + length) % length;
+            state.context.selectedKey = state.info.enabledKeys[newCurrTabItemIndex];
+            data.onKeySelect(state.context.selectedKey);
+          }
+        }
+      }
+    };
+
+    /* GH #964, Extra props are needed because FocusZone is not implemented on windows.
+    The ref focusZoneRef is used to set focus on Tabs when selecting a TabsItem and onKeyDown manages keyboarding */
+    const slotProps = mergeSettings<TabsSlotProps>(styleProps, {
+      root: { ref: componentRef, accessibilityLabel: accessibilityLabel, accessibilityRole: 'tablist', ...pressable.props, ...rest},
+      label: { children: label },
+      stack: { focusable: true, ref: focusZoneRef, onKeyDown: onKeyDown},
+    });
+
+    return { slotProps, state };
+  },
+
+  render: (Slots: ISlots<TabsSlotProps>, renderData: TabsRenderData, ...children: React.ReactNode[]) => {
+    if (!renderData.state) {
+      return null;
+    }
+
+    // Populate the tabsItemKeys array
+    if (children) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore - TODO, fix typing error
+      renderData.state.context.tabsItemKeys = React.Children.map(children, (child: React.ReactChild) => {
+        if (React.isValidElement(child)) {
+          // Sets default selected tabItem
+          if (renderData.state.context.selectedKey == null && !child.props.disabled) {
+            renderData.state.context.selectedKey = child.props.itemKey;
+          }
+          return child.props.itemKey;
+        }
+      });
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore - TODO, fix typing error
+      renderData.state.info.enabledKeys = React.Children.map(children, (child: React.ReactChild) => {
+        if (React.isValidElement(child)) {
+          if (!child.props.disabled) {
+            return child.props.itemKey;
+          }
+        }
+      });
+    }
+
+    return (
+      <TabsContext.Provider
+        // Passes in the selected key and a hook function to update the newly selected tabsItem and call the client's onTabsClick callback
+        value={renderData.state.context}
```

Comment:
`renderData.state?.context` You check for `!renderData.state` above, but that doesn't mean renderData.state.context != nullptr